.github/workflows/e2e-k8s.yml

```diff
@@ -70,7 +70,7 @@ jobs:
       - name: Create k8s Kind Cluster
         run: |
           # install kubectl
-          curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
```

Comment:
But I can get `v1.28.3` with command `echo $(curl -L -s https://dl.k8s.io/release/stable.txt)`

Reply:
I think the reason @Gallardot wants to check the latest stable version, so it use value from this link

Reply:
I understand the purpose of this PR. It is indeed necessary to use a specific version of kubectl instead of using the latest version of kubectl because the iteration of k8s version is relatively frequent.  Perhaps we should unify the versions of kubectl and the kind cluster to avoid version differences.

@ruanwenjun @zhongjiajie

Reply:
Yes, and this change is due to current our ci has blocked, this might due to we get the incompatibility version.